src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#define DEBUG
```

Comment:
[clarification] Why is this DEBUG needed?

Reply:
My bad. Forgot to remove it. This tag basically allows logging to output window using `Debug.Writeline` in Release mode.